test/python/test_with_schedules.py

```diff
@@ -387,6 +414,12 @@ def test_prdelta_parallel_load_balance_hybrid_dense_without_bitvec(self):
 
     def test_eigenvector_centrality(self):
 	self.eigenvector_centrality_verified_test("eigenvector_centrality.gt", True)
+       
+    def test_closeness_centrality_unweighted(self):
```

Comment:
You need to rename the two schedule files (closeness_centrality_unweighted.gt for example). The schedule files should indicate what kind of schedule is being used. Take a look at the other schedule files as examples. You usually need to note the parallelization strategies and directions in the name of the schedule files. The name of the tests should be renamed to indicate the schedule you are testing as well. Take a look at the names of the other unit tests.